### PR TITLE
Adding input files for all three cloud formation resources

### PR DESCRIPTION
**Pull request by chrigilm**
*Description of changes:*
Adding the three input files per cloud formation resource in order for the contract tests to be able to run during registration.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-globalaccelerator-endpointgroup/README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ There are a couple options for building and testing the project.
 ##### Files:
 
 1. `aws-globalaccelerator-endpointgroup.json` - This file contains the schema definition for the endpoint group resource.  It also drives auto generated classes from rpdk.
+2. `inputs/inputs_1_create.json` - This file is used during the contract tests which are run when using the "cfn submit" or "cfn test" commands.  When running either of those commands, replace {{ListenerArn}} in this file with a valid Listener Arn from one of your Global Accelerators in your aws account.
```
Comment (DJ-C) [inline aws-globalaccelerator-endpointgroup/README.md:13]: nit: is it possible/easy to remove "_1_" from input file name?

File: aws-globalaccelerator-accelerator/inputs/inputs_1_create.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (DJ-C) [inline aws-globalaccelerator-accelerator/inputs/inputs_1_create.json:1]: Can we add README,md for accelerator resource as well?

**Review by DJ-C — COMMENTED** [review]
Left some nit comments.

File: aws-globalaccelerator-endpointgroup/README.md (hunk 1 above)
Comment (chrigilm) [reply aws-globalaccelerator-endpointgroup/README.md:13]: No, this is a requirement of the contract tests.  It allows for multiple versions of the same type (create, invalid, or update)

Comment (DJ-C) [reply aws-globalaccelerator-endpointgroup/README.md:13]: Thanks for clarification.

File: aws-globalaccelerator-accelerator/inputs/inputs_1_create.json (hunk 2 above)
Comment (chrigilm) [reply aws-globalaccelerator-accelerator/inputs/inputs_1_create.json:1]: Whoops,  missed being part of commit.  Fixed

**Review by DJ-C — APPROVED** [review]

**Comment by DJ-C** [comment]
Code build is failing so you may want to fix. Otherwise, looks great.

**Pull request merged**